recipes/esparto/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "esparto" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/esparto-{{ version }}.tar.gz
+  sha256: 42983f2f83cd0a1c83cd336a6c4a73ea032e10c1cbc01edc1399080183b05691
+
+build:
+  number: 0
+  skip: true   # [py>=40 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry >=0.12
+    - python
+  run:
+    - dataclasses  # [py<37]
```

As this is harmless to include for all versions of Python and enables the `noarch: python` benifits:
```suggestion
    - dataclasses
```